### PR TITLE
Igor check/update for storage.rst.

### PR DESCRIPTION
**Pull request by a-stjohn**
This commit is a master port for #331 . I ended up finding and correcting a lot of other style fallacies. Thanks igor.

### CODE REVIEW

File: userguide/storage.rst [hunk 1]
```diff
@@ -734,10 +734,10 @@ clicking the :guilabel:`Edit Options` button in
 After a dataset is created, click on that dataset and select
 :guilabel:`Create Dataset`, to create a nested dataset, or a
 dataset within a dataset. A zvol can also be created within a dataset.
-When creating datasets, double-check that you are using the
-:guilabel:`Create Dataset` option for the intended volume or dataset.
-If you get confused when creating a dataset on a volume, click all
-existing datasets to close them--the remaining
+When creating datasets, ensure that the :guilabel:`Create Dataset`
+option is being used for the intended volume or dataset.
```
Comment (wblock) [inline userguide/storage.rst]: These two sentences seem to be saying that the GUI is confusing when creating a dataset.  It probably is, but do what you can to make this explanation clear.

File: userguide/storage.rst [hunk 2]
```diff
@@ -2703,11 +2698,11 @@ corruptions caused by transient hardware issues, and provide early
 alerts of impending disk failures. %brand% makes it easy to schedule
 periodic automatic scrubs.
 
-Each volume should be scrubbed at least once a month. Bit errors in
-critical data can be detected by ZFS, but only when that data is read.
-Scheduled scrubs can find bit errors in rarely-read data. The amount
-of time needed for a scrub is proportional to the quantity of data on
-the volume. Typical scrubs take several hours or longer.
+It is recommended that each volume be scrubbed at least once a month.
```
Comment (wblock) [inline userguide/storage.rst]: Avoid "It is recommended." This sentence can be rearranged as a recommendation:
"Each volume should be scrubbed at least once a month."
(Note: "should" is used as a recommendation here.)

File: userguide/storage.rst (hunk 1 above)
Comment (a-stjohn) [reply userguide/storage.rst]: I cleaned it up some. However, I'm sure it could be better?

File: userguide/storage.rst [hunk 3]
```diff
@@ -731,14 +733,11 @@ clicking the :guilabel:`Edit Options` button in
...
-dataset within a dataset. A zvol can also be created within a dataset.
-When creating datasets, ensure that the :guilabel:`Create Dataset`
-option is being used for the intended volume or dataset.
-When creating a dataset on a volume, click all
-existing datasets to close them. The remaining
-:guilabel:`Create Dataset` will be for the volume.
+Create a nested dataset by clicking on an existing dataset and selecting
+:guilabel:`Create Dataset`. A zvol can also be created within a dataset.
+Ensure that the :guilabel:`Create Dataset` option is being used for
+the intended volume or dataset by collapsing it. Collapse the volume or
```
Comment (wblock) [inline userguide/storage.rst]: Nonspecific "it" here--does it mean the volume or the dataset?

File: userguide/storage.rst [hunk 4]
```diff
@@ -731,14 +733,11 @@ clicking the :guilabel:`Edit Options` button in
...
+dataset by clicking the black trianlge beside the name.
```
Comment (wblock) [inline userguide/storage.rst]: Tyop.

**Pull request merged**